modules/Bio/EnsEMBL/Variation/TranscriptVariationAllele.pm

```diff
@@ -1417,15 +1417,15 @@ sub hgvs_transcript {
   my @edit_attrs = grep {$_->code =~ /^_rna_edit/} @attribs;
 
   my $misalignment_offset = 0;
-  $misalignment_offset = $self->get_misalignment_offset(\@edit_attrs) if (scalar(@edit_attrs) && substr($tr->stable_id, 0,3) eq 'NM_');
+  $misalignment_offset = $self->get_misalignment_offset(\@edit_attrs) if (scalar(@edit_attrs) && (substr($tr->stable_id, 0,3) eq 'NM_' || substr($tr->stable_id, 0,3) eq 'XM_'));
```

Comment:
The example is NM, is the addition of `XM_` another bug that is fixed in this PR? If so, could this please be added in the PR comment. Thanks.

Reply:
Thanks, comment added